src/Components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="buttons-div">
+      <div className="input-field">0</div>
+    </div>
+    <div className="buttons-div">
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button" className="operation">÷</button>
+    </div>
+    <div className="buttons-div">
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button" className="operation">x</button>
+    </div>
+    <div className="buttons-div">
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
+      <button type="button" className="operation">-</button>
```

Comment:
- [ ] Good work up to this point. However, it looks like your design was affected after working on the changes requested. Kindly check it out

![image](https://github.com/kkmanuu/math-magicians/assets/98453979/e9ad6164-8fa3-435a-a462-5ef1e7123473)

Reply:
@frank1738 I have implemented the required changes when you inspect the design looks awesome as the required changes you mention.